azurerm/resource_arm_traffic_manager_endpoint.go

```diff
@@ -110,6 +110,28 @@ func resourceArmTrafficManagerEndpoint() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"subnets": {
```

Comment:
can we rename this `subnet`, since users will specify this as `subnet {}` in HCL?

```suggestion
			"subnet": {
```

can we add this new block [to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/traffic_manager_endpoint.html.markdown#L82)?

Since there's no example of a sub-block there; we'd generally format this as:

```
* `subnet` - (Optional) One or more `subnet` blocks as defined below

[other fields]

---

A `subnet` block supports the following:

* `first` - (Required) The First IP....

* `last` - (Optional) The Last IP...

* `scope` - (Optional) The Scope...

-> **NOTE:** One of either `last` or `scope` must be specified
```

Reply:
Thank you for your comment. However, it seems to be better to keep the name as "subnets" to corresponds with this document. 
[Microsoft.Network trafficmanagerprofiles template reference] (https://docs.microsoft.com/en-us/azure/templates/microsoft.network/2018-04-01/trafficmanagerprofiles)

By the way, the "subents" is a list of one or more subnet templates, so keeping the name seems nice.

Reply:
Since we're expecting most users to define them as individual blocks (for example):

```
subnet {
  ...
}

subnet {
  ...
}
```

.. we're moving towards a convention of using the singular name (e.g. `subnet` rather than `subnets`) throughout the Provider. Whilst we do look towards the Azure API/SDK for naming inspiration - in places such as this we tend to err on the side of using the singular naming rather than the pluralised name - as such could we update this to `subnet`?

Reply:
Ok, I changed the subnets to subnet.